auditbeat/magefile.go

```diff
@@ -27,11 +27,11 @@ import (
 	"github.com/magefile/mage/mg"
 
 	auditbeat "github.com/elastic/beats/auditbeat/scripts/mage"
-	"github.com/elastic/beats/dev-tools/mage"
+	devTools "github.com/elastic/beats/dev-tools/mage"
 )
 
 func init() {
-	mage.BeatDescription = "Audit the activities of users and processes on your system."
+	devTools.BeatDescription = "Audit the activities of users and processes on your system."
```

Comment:
Referring to a package with a mixedCaps name is unusual. It makes me think this is a global variable. Check out the recommendations in https://golang.org/doc/effective_go.html#package-names.